packages/teleterm/src/ui/DocumentGateway/common.tsx

```diff
@@ -25,8 +25,21 @@ export const ConfigFieldInput: typeof FieldInput = styled(FieldInput)`
     }
 `;
 
+const ConfigFieldInputWithoutStepper: typeof ConfigFieldInput = styled(
+  ConfigFieldInput
+)`
+  ::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
+  ::-webkit-outer-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
+`;
+
 export const PortFieldInput: typeof ConfigFieldInput = props => (
-  <ConfigFieldInput
+  <ConfigFieldInputWithoutStepper
     type="number"
```

Comment:
Many answers on the internet recommend using `type="text" inputmode="numeric"` if you don't need the stepper. But if we use that, we lose all the nice validations that come with `type="number"` + it's possible to type any character on a regular keyboard.

I decided to hide the stepper then and categorize this as lack of browser's support for hiding the stepper through another attribute. 😏

Reply:
Totally agree!